src/less/chips-combobox/chips-combobox.less

```diff
@@ -0,0 +1,113 @@
+@import "../variables/variables.less";
+
+.chips-combobox {
+    background-color: var(--color-background-secondary);
+    border: 1px solid var(--color-stroke-default);
+    border-radius: var(--border-radius-50);
+    padding-bottom: var(--spacing-100);
+    padding-inline-end: var(--spacing-200);
+    padding-inline-start: var(--spacing-100);
+    padding-top: var(--spacing-100);
+    position: relative;
+}
+
+.chips-combobox__items:empty {
+    display: none;
+}
+
+.chips-combobox__items:empty ~ .chips-combobox__combobox {
+    margin-top: 0;
+}
+
+.chips-combobox .combobox__control > input {
+    background-color: inherit;
+    border: none;
+    border-radius: 0;
+    height: auto;
+    margin: 0 var(--spacing-100);
+    padding: var(--spacing-75) 0;
+    width: calc(100% - var(--spacing-200));
+}
+
+.chips-combobox .combobox__control > input:focus {
+    outline: none;
+}
+
+.chips-combobox .combobox {
+    margin-inline-start: calc(var(--spacing-100) * -1);
+    margin-top: var(--spacing-100);
+    width: calc(100% + var(--spacing-300));
+}
+
+.chips-combobox .combobox svg.icon {
+    pointer-events: none;
+    position: absolute;
+    right: var(--spacing-200);
+    top: calc(50% - var(--spacing-100));
+    transform: rotateX(0deg);
```

Comment:
I think this vertical flip is awesome, but we don't do it anywhere else so we should probably ask design about it first. Regular combobox also uses `rotate` instead of `rotateX` so it would be a spin instead of a flip if we added `transition`.

Reply:
The regular combobox actually uses 12px chevron. I'm assuming this will be easily swappable in ebay ui to the 16px chevron that's being used here and per DSs specs. The `css` transforms for combobox are specific to the 12px icon so it would not interfere with this.

I'd actually prefer to keep this, if possible. We can roll it out, demo for design, and if they don't like it, removing the transform and transition is easily doable without a breaking change.